src/librustc_llvm/lib.rs

```diff
@@ -24,10 +24,12 @@
 #![feature(link_args)]
 #![feature(static_nobundle)]
 
-extern crate libc;
+#[allow(unused_extern_crates)]
```

Comment:
Can you add a comment here pointing to the `Cargo.toml` of rustc_cratesio_shim?

Reply:
Done.